### PR TITLE
FarhanNurzi/WALL-608/DIEL - Show deposit options

### PR DESCRIPTION
**Pull request by farhan-nurzi-deriv**
Screenshot 2023-07-25 at 3 37 08 PM
Screenshot 2023-07-25 at 2 48 29 PM

### CODE REVIEW

File: packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import useLiveChat from 'App/Components/Elements/LiveChat/use-livechat';
+
+const OneTimeDepositModal = () => {
+    const { data: iframe_url, isSuccess } = useDepositFiatAddress();
+    const [is_iframe_loading, setIsIframeLoading] = useState(true);
+    const { client, ui } = useStore();
+    const { has_deposited_for_first_time, loginid } = client;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx:23]: has_deposited_for_first_time was just created for testing purpose. we need to find a better way to trigger this modal

File: packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    Modal,
+    Text,
+} from '@deriv/components';
+import { useDepositFiatAddress } from '@deriv/hooks';
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import useLiveChat from 'App/Components/Elements/LiveChat/use-livechat';
+
+const OneTimeDepositModal = () => {
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx:19]: const OneTimeDepositModal = observer() => {
.....

}


export default OneTimeDepositModal

File: packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        @include mobile {
+            width: 100%;
+            padding: 2rem;
+        }
+    }
+
+    &__description {
+        display: grid;
+        grid-template-columns: 78fr 22fr;
+        gap: 0.5rem;
```
Comment (shaheer-deriv) [inline packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.scss:31]: is 0.5rem the right value as per figma design file?

Comment (farhan-nurzi-deriv) [reply packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.scss:31]: let me re-check it 🙏

**Review by ameerul-deriv — APPROVED** [review]

**Review by shaheer-deriv — APPROVED** [review]

**Review by niloofar-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**